src/tests/suite/colorize-fixtures/createGlobalStyles.js

```diff
@@ -0,0 +1,5 @@
+const GlobalStyles = createGlobalStyle`
```

Comment:
Hey @josiahwiebe should this be `createGlobalStyles` ?
Also the tests are failing, I'm not sure why. It could be something has changed since you last generated the fixture